src/api/treeTrackerApi.js

```diff
@@ -163,16 +163,27 @@ export default {
    */
   getSpecies(abortController) {
     const query = `${process.env.REACT_APP_API_ROOT}/api/species`;
-    return fetch(query, {
-      method: 'GET',
-      headers: {
-        'content-type': 'application/json',
-        Authorization: session.token,
-      },
-      signal: abortController?.signal,
-    })
-      .then(handleResponse)
-      .catch(handleError);
+    return (
+      fetch(query, {
+        method: 'GET',
+        headers: {
+          'content-type': 'application/json',
+          Authorization: session.token,
+        },
+        signal: abortController?.signal,
+      })
+        .then(handleResponse)
+        // Sort species list alphabetically by species name
+        .then((res) => {
+          res = res.sort((a, b) => {
```

Comment:
Our API actually supports ordering in the query string, so there's no need to sort the results in the client:
```js
165:    const query = `${process.env.REACT_APP_API_ROOT}/api/species?filter[order]=name`;
```

Reply:
Thank you for your insight! I removed the client-side filtering, made the change to the query that you suggested, and merged the upstream changes from master into this branch